src/Analyzer/ClosureAnalyzer.php

```diff
@@ -55,14 +55,15 @@ private function determineBinding(array &$data)
 
     private function isClosureStatic(\Closure $closure)
     {
-        $closure = @$closure->bindTo(new \stdClass);
+        if (version_compare(PHP_VERSION, '7.0', '>=')) {
+            $closure = @$closure->bindTo(new \stdClass);
 
-        if ($closure === null) {
-            return true;
+            if ($closure === null) {
+                return true;
+            }
         }
-
+        
         $rebound = new \ReflectionFunction($closure);
-
```

Comment:
Please restore this newline, and remove the whitespace on the line above, then this looks good, thank you!